astroquery/linelists/cdms/core.py

```diff
@@ -277,7 +277,8 @@ def _parse_result(self, response, *, verbose=False):
 
         return result
 
-    def get_species_table(self, *, catfile='catdir.cat'):
+    def get_species_table(self, *, catfile='catdir.cat', use_cached=True,
+                          catfile_url='https://cdms.astro.uni-koeln.de/classic/entries/partition_function.html'):
```

Comment:
I wonder whether this should be in the config, or here?

Reply:
probably config